src/test/java/Helper/DateUtil.java

```diff
@@ -45,4 +75,30 @@ public static boolean isDateInCorrectFormat(String dateText, String expDateForma
         return isDateFormatCorrect;
     }
 
-}
+
+
+    public static List<String> textDateToDateFormat(List dateText, String expDateFormat) {
+        SimpleDateFormat sdf = new SimpleDateFormat(expDateFormat);
+        boolean isDateFormatCorrect = true;
+        List<String> newList = dateText;
```

Comment:
dateText is already a List type data, not sure why you storing it again into List type data.